### ISSUE TITLE
wrong output for language detection model 

### ISSUE THREAD

**Issue by amandalmia14** [facebookresearch/fastText#1181]
Hi Team,

When I tries to predict the language detection model for the below sentence I am not sure why its giving me "Korean" language as an output. 

model.predict("PROJECT TAGGING CRITERIA")
-> (('__label__ko',), array([0.06832896]))

**Comment by DileepkumarT** [comment]
lower the case and try ...i hope it will work. And predictions are not that great for short text

**Comment by amandalmia14** [comment]
@DileepkumarT Yes, it will work, if do lowercase, however my dataset is multilingual. I am not sure how it will behave for other languages.
